  Omit FLTK-dependent utilities for Hammerfall
  Omit qlo10k1 utility (GUI for ld10k1); it has apparently broken configure.in
--- alsa-tools-1.0.22.orig/Makefile	2010-01-09 21:20:38.402448662 +0100
+++ alsa-tools-1.0.22/Makefile	2010-01-09 21:20:48.709118174 +0100
@@ -1,8 +1,8 @@
 VERSION = 1.0.22
 TOP = .
-SUBDIRS = ac3dec as10k1 envy24control hdsploader hdspconf hdspmixer \
+SUBDIRS = ac3dec as10k1 envy24control \
 	  mixartloader pcxhrloader rmedigicontrol sb16_csp seq sscape_ctl \
-	  us428control usx2yloader vxloader echomixer ld10k1 qlo10k1
+	  us428control usx2yloader vxloader echomixer ld10k1
 
 all:
 	@for i in $(SUBDIRS); do \
